modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -333,12 +333,44 @@ definition {
 	}
 
 	@description = "Verify that the user can order the results inside Assigned to My Roles by Due Date"
-	@ignore = "Test Stub"
 	@priority = "2"
 	test CanOrderAssignedtoMyRolesTasksByDueDate {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry 1");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry 2");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.updateDueDateTaskByActions(
+			workflowAssetTitle = "Blogs Entry 2",
+			workflowAssetType = "Blogs Entry",
+			workflowDueDateTime = "1:45 PM",
+			workflowTask = "Review");
 
-		// TODO LRQA-68840 CanOrderAssignedtoMyRolesTasksByDueDate pending implementation
+		Notifications.goToFilterValues(filterValues = "Due Date");
 
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry 2",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_DUE_DATE",
+			value1 = "Within 8 Months");
```

Comment:
Try to create macros from these functions when you feel the need to use them in `.testcase `files. Complementing the  comment above, ideal is to use a macro to ensure the position of items, you can replace this piece and use the same _macro_ `viewMyWorkflowTaskItemPosition` and _path_ `MY_WORKFLOW_TASK_ITEM_POSITION` that you have created at PR #843. Thank you!